templates/shortcodes/notice_box.md

```diff
@@ -0,0 +1,3 @@
+<div style="text-align: center; padding: .4em 1em; margin-bottom: 1rem; background-color: #ffcb006b;">
+{{ body }}
```

Comment:
```suggestion
{{ body | markdown | safe }}
```

Should fix the markdown rendering not working

Reply:
works, done!